.circleci/wait-for-job-finish.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Exit script if you try to use an uninitialized variable.
+set -o nounset
+
+# Use the error status of the first failure, rather than that of the last item in a pipeline.
+set -o pipefail
+
+x=0
+EXCLUDE_NODE_INDEX=0
```

Comment:
Could this variable be replaced with `CIRCLE_NODE_INDEX`? I think that might make it clearer that it's waiting for all other nodes to finish.

Reply:
Yeah, updated `jq` selector to use it: `select(.index != '$CIRCLE_NODE_INDEX')'`